feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-bundle/configurator-attribute-multi-selection-bundle.component.html

```diff
@@ -2,10 +2,18 @@
   id="{{ createAttributeIdForConfigurator(attribute) }}"
   *ngIf="attribute?.values?.length"
 >
+  <cx-configurator-attribute-quantity
```

Comment:
please wrap this component in `<div>` similar to single-selection-bundle component. also add price-related div.

Reply:
done